panther_docking/launch/docking.launch.py

```diff
@@ -62,6 +69,7 @@ def generate_launch_description():
             namespaced_docking_server_config,
             {"use_sim_time": use_sim},
         ],
+        arguments=["--ros-args", "--log-level", log_level, "--log-level", "rcl:=INFO"],
```

Comment:
It seems redundant
```suggestion
        arguments=["--ros-args", "--log-level", log_level],
```

Reply:
`, "--log-level", "rcl:=INFO"` is added to set log level for `rcl` always to `INFO`. 

So when it is added and `log_level` set to `debug` you don't see `rcl` debug logs what are really spammyy.

Without `rcl` DEBUG
![image](https://github.com/user-attachments/assets/00a6c046-f7da-4cdb-b04d-f9b6845e56cd)

With `rcl` DEBUG
![image](https://github.com/user-attachments/assets/f2da31e5-3ed2-43b3-b8b3-89bfdeafc2e5)
And  in the runtime:
![image](https://github.com/user-attachments/assets/27034e33-510c-4d59-b408-51a65416781e)

Reply:
@rafal-gorecki  what about that?